src/stratigraphy/benchmark/score.py

```diff
@@ -45,6 +71,8 @@ def evaluate_matching(predictions_path: Path, ground_truth_path: Path) -> tuple[
     with open(predictions_path) as in_file:
         predictions = json.load(in_file)
 
+    draw_predictions(predictions, ground_truth, directory, out_directory)
```

Comment:
We could take this one step further, and instead of calling `ground_truth.is_correct()` for every prediction twice (once when drawing and once when computing the scores), we could compute the truth value for every prediction in advance, and then reuse it for both drawing and computing the scores.